test/e2e/monitoring_stack_controller_test.go

```diff
@@ -577,29 +578,11 @@ func prometheusScaleDown(t *testing.T) {
 
 	err := f.K8sClient.Create(context.Background(), ms)
 	assert.NilError(t, err, "failed to create a monitoring stack")
-
-	prom := monv1.Prometheus{}
-	f.GetResourceWithRetry(t, ms.Name, ms.Namespace, &prom)
-
-	assert.Equal(t, prom.Status.Replicas, int32(1))
+	f.AssertPrometheusReplicaStatus(ms.Name, ms.Namespace, numOfRep)
 
 	err = f.UpdateWithRetry(t, ms, framework.SetPrometheusReplicas(0))
-	key := types.NamespacedName{Name: ms.Name, Namespace: ms.Namespace}
 	assert.NilError(t, err, "failed to update a monitoring stack")
-	err = wait.PollUntilContextTimeout(context.Background(), 5*time.Second, framework.DefaultTestTimeout, true, func(ctx context.Context) (bool, error) {
-		if err := f.K8sClient.Get(context.Background(), key, &prom); errors.IsNotFound(err) {
-			return false, nil
-		}
-
-		if prom.Status.Replicas != 0 {
-			return false, nil
-		}
-		return true, nil
-	})
-
-	if wait.Interrupted(err) {
-		t.Fatal(fmt.Errorf("Prometheus was not scaled down"))
-	}
+	f.AssertPrometheusReplicaStatus(ms.Name, ms.Namespace, numOfRep)
```

Comment:
```suggestion
	f.AssertPrometheusReplicaStatus(ms.Name, ms.Namespace, 0)
```
Should this test for scale down to 0?

Reply:
Sorry typo, thanks your sharp eyes